docs/gsoc_2019/proposal.md

```diff
@@ -0,0 +1,86 @@
+# New features for Google Summer of Code 2019
```

Comment:
I'd suggest changing the folder to `kap_proposals` from `gsoc_2019` and add each of these features are a separate doc e.g. `kap_0_external_dependencies.md` .. etc. In the same style as python does it https://www.python.org/dev/peps/ 
Then we can add more of these in the future. :) One for Kadet for example would be needed.

Reply:
Of course you can add a link to your full proposal and your name as `Author: .... ` to each of these :) Same as in the PePs.

Reply:
Got it!